frontend/src/App.js

```diff
@@ -1,15 +1,45 @@
 import React, {useState, useEffect} from 'react';
+import dotenv from "dotenv";
 import './App.css';
+import GitHubLogin from 'react-github-login';
```

Comment:
Please do not use this package. It is not maintained, and not widely used. Thank you.